DataTypes/Program.cs

```diff
@@ -1,12 +1,166 @@
+            creature = Console.ReadLine();
+
+            if (creature == "y")
+                
+            {
+                Console.WriteLine();
+                Thread.Sleep (3000);
+                Console.WriteLine("As you move closer you realize that the glowing object is an enourmous eye.");
+                Console.WriteLine();
+                Thread.Sleep (3000);
+                Console.WriteLine("You are now face to face with a gigantic creature!");
+            }
+            
+            else if (creature == "n")
+            {
+                Console.WriteLine();
+                Console.WriteLine("You try to slip by the object without disturbing it, as you slip by you notice the object starts to move!");
+                Console.WriteLine();
+                Thread.Sleep (3000);
+                Console.WriteLine("The glowing object turns to you! You realize you're staring into the eye of a gigantic creature!");
+                Thread.Sleep (3000);
+            }
+
+            Console.WriteLine();
+            Thread.Sleep (3000);
+            Console.WriteLine("Your first instinct is to run for yor life, do you turn and fight the creature instead?  y/n");
+            fight = Console.ReadLine();
+            //fight is unavoidable//
+            //with lit torch odds are better//
+            if (fight == "y" & walk =="y")
+            {
+                Random r = new Random();
+                int number = r.Next(3, 10);
+
+                if (number < 4)
+                {
+                    Console.WriteLine("You thrust feebly at the creature but miss and stumble forward. The creature pounces and sinks it's fangs into your back, you lay paralzyed and await death.");
+                    Console.WriteLine();
+                }
+                else if (number < 7)
+                {   Thread.Sleep (3000);
+                    Console.WriteLine("You fight the creature valiantly but you just don't have a good enough weapon.  Eventually the creature wears you down and grabs you, it then begins wrapping you in it's deathly grip");
+                    Console.WriteLine();
+                }
+                else if (number < 9)
+                {   Thread.Sleep (3000);
+                    Console.WriteLine("You get a clean strike at the creatures neck you hear a nice sizzle the creature shrieks and runs away!");
+                    Console.WriteLine();
+                }   
+                else 
+                {   Thread.Sleep (3000);
+                    Console.WriteLine("You get a perfect strike and puncture the creatures neck and it shirvelled up and died. There is a nice pile of loot that the creature dropped from it's previous victims. Your pockets are filled with gold and jewels as you escape the forest!");
+                    Console.WriteLine();
+                }
+                   
+            }
+            else if (fight == "y" || walk == "n")
+            {
```

Comment:
You could trim some code down here by setting int number within one if/else and only having the dice roll possibilities written once. 
For example:

int num;
Random r = new Random();
if(fight == "y" && walk =="y")
{
   num =  r.Next(3,10);
}
else if(fight == "y" && walk =="n")
{
   num = r.Next(1,9);
}
// Here you'd only have to write out if(num <4) and so on once